.github/workflows/main.yml

```diff
@@ -9,6 +9,11 @@ jobs:
   build:
     name: Build and Publish SDKs
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
Do we need to explicitly set permission values because setting `id-token` (defaults to `none`) will reset the default values of the other properties? What actions are we using that require these permissions (ie. `issues`)?

Reply:
This was the config suggested in the documentation of semantic-release. Removed the ones we don't need